src/main/java/com/launchdarkly/client/StreamProcessor.java

```diff
@@ -227,23 +227,16 @@ public DeleteData() {
   private class DefaultEventSourceCreator implements EventSourceCreator {
     public EventSource createEventSource(EventHandler handler, URI streamUri, ConnectionErrorHandler errorHandler, Headers headers) {
       EventSource.Builder builder = new EventSource.Builder(handler, streamUri)
+          .client(config.httpClient)
           .connectionErrorHandler(errorHandler)
           .headers(headers)
           .reconnectTimeMs(config.reconnectTimeMs)
-          .connectTimeoutMs(config.connectTimeoutMillis)
           .readTimeoutMs(DEAD_CONNECTION_INTERVAL_MS);
       // Note that this is not the same read timeout that can be set in LDConfig.  We default to a smaller one
       // there because we don't expect long delays within any *non*-streaming response that the LD client gets.
       // A read timeout on the stream will result in the connection being cycled, so we set this to be slightly
       // more than the expected interval between heartbeat signals.
 
-      if (config.proxy != null) {
```

Comment:
setting the proxy here isn't needed anymore, since it'll be copied over from the client in the config